minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/logback.xml

```diff
@@ -29,12 +29,14 @@
               To ZIP rolled files, replace '.log' with '.log.zip'.
             -->
             <fileNamePattern>${org.apache.nifi.minifi.bootstrap.config.log.dir}/${org.apache.nifi.minifi.bootstrap.config.log.app.file.name}_%d{yyyy-MM-dd_HH}.%i.${org.apache.nifi.minifi.bootstrap.config.log.app.file.extension}.gz</fileNamePattern>
-            <!-- Keep 10 rolling periods worth of log files-->
-            <maxHistory>10</maxHistory>
-            <!-- Max size each log file will be-->
+            <!-- Control the maximum size of each log file before rolling over -->
             <maxFileSize>1MB</maxFileSize>
-            <!-- Provide a cap of 10 MB across all archive files -->
+            <!-- Control the maximum number of log archive files kept and asynchronously delete older files -->
+            <maxHistory>10</maxHistory>
+            <!-- Control the total size of all log archive files for this appender -->
             <totalSizeCap>10MB</totalSizeCap>
+            <!-- Log files exceeding maximum settings will be rolled on startup -->
+            <cleanHistoryOnStart>true</cleanHistoryOnStart>
```

Comment:
While we're making changes in this file, I'd be in favor for increasing 1MB to 10MB and 10MB to 100MB as the current values seem a bit small to me.

Reply:
Thanks @pvillard31, I was also thinking those defaults were on the small side. I pushed a change for this MiNiFi configuration to use `maxFileSize` of 10MB and `totalSizeCap` of 100MB.